<launch>
   <node name="vel_controller" pkg="lab01" type="vel_controller_node" output="screen">
      <param name="kp_lin" value = "0.9" />
      <param name="ki_lin" value = "0.1" />
      <param name="kd_lin" value = "0.05" />
      <param name="kp_ang" value = "1.0" />
      <param name="ki_ang" value = "0.0" />
      <param name="kd_ang" value = "0.0" />
   </node>
</launch>
